modules/deploy/pages/deployment-option/cloud/api/cloud-api-overview.adoc

```diff
@@ -0,0 +1,61 @@
+= Redpanda Cloud API Overview
+:description: Learn about using the Cloud API to manage clusters and cloud resources.
+:page-cloud: true
+:page-api: cloud
+:page-api-slot: overview
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management and manage Redpanda Cloud resources.
+
+== Requirements
+
+To be able to use the Cloud API:
+
+* You must be a customer with an existing organization in Redpanda Cloud.
+* You can only use one organization for authentication. See xref:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[Cloud API Authentication] for steps to authenticate API requests.
```

Comment:
Maybe rephrase this a bit, e.g. to explain that a token is always for a specific organization. if they want to switch to a different organization, they must acquire a new token (with a service account from the other org)

Reply:
@birdayz Added in [this commit](https://github.com/redpanda-data/docs/pull/411/commits/4bf8f7c15d7e5d703b724595b5fb4cd2264f58c3); please let me know if the change doesn't address this!